src/v/cluster/controller_backend.cc

```diff
@@ -1115,6 +1182,11 @@ controller_backend::process_partition_reconfiguration(
               std::move(ntp), command_rev);
         }
 
+        if (type == topic_table_delta::op_type::force_update) {
+            // TODO: Add a detailed comment.
+            co_return errc::success;
+        }
```

Comment:
why we do not want to force reconfigure partition in this case i.e. it exists on current shard but should not ? i think we should, imagine a situation in which replicas become alive after some time